content/get-started/index.md

```diff
@@ -0,0 +1,78 @@
+---
+title: Get Started
+---
+<a href="https://galaxy-mentor-network.netlify.app/" class="btn btn-primary" role="button" aria-pressed="true">Would you like to learn about Galaxy with a mentor? Check out the Galaxy Mentorship Network</a>
+
+## Welcome!
+
+Below you will find tutorials, useful links and definitions as the starting point for newcomers,students, experts and anybody who wants to refresh their knowledge about Galaxy's basics and capabilities.
+
+### Galaxy Glossary
+[List](link) of all keywords used within Galaxy and their definitions.
```

Comment:
It should point to the glossary, right?

I would also introduce a bit the "why" for the glossary. Maybe something like (and sorry I am not very good at making sentences...):

*The Galaxy glossary contains all the terminology, acronyms and their definitions. As a newbee, you will probably skip many of its content but later on in your Galaxy journey, you will come back to it and it will help you to get a common understanding of the Galaxy ecosystem.*

Reply:
Yes it should! 

I like the sentence, maybe 

_The Galaxy glossary contains all the terminology, acronyms and their definitions. As a newbie, you will probably skim through this list but later on in your Galaxy journey you will probably come back to refer to it as it will help you get a better understanding of the Galaxy Ecosystem._